packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -181,6 +181,13 @@ export interface SubnetConfiguration {
      * availability zone.
      */
     name: string;
+
+    /**
+     * The AWS resource tags to associate with the Subnet.
+     *
+     */
+    tags?: cdk.Tag[];
```

Comment:
Everywhere I exposed to the customer I tried to use `cdk.Tag` but internally I need to use `cdk.Token` to get the lazy evaluation. I'm not sure if this is the right pattern or not?

Reply:
Wouldn't it make more sense for this to just be `{ [key: string]: any }`?

Reply:
I think we should deprecate `cdk.Tag`. Not sure what value it brings to this world...

Reply:
@eladb and the second question here about `{ [key: string]: string } ` vs `any`?